bots/api/bot.py

```
@@ -37,19 +75,106 @@ async def startup_event():
+
+        return {"message": "success", "deviceToken": jwt_token}
+
+    except HTTPException as e:
+        raise e
+    except Exception as e:
+        Logger.error(str(e))
+        raise HTTPException(status_code=400, detail="bad request")
+
+
+@app.post("/verify-token")
+async def verify_token(credentials: HTTPAuthorizationCredentials = Depends(HTTPBearer())):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)